### PR TITLE
get_r_all implementations for multislot estimator

### PR DESCRIPTION
**Pull request by ataymano**
estimatior (point/interval) of average reward from all slots

### CODE REVIEW

File: estimators/test/utils.py [hunk 1]
```diff
@@ -39,11 +47,20 @@ def get_intervals(scenarios):
 def get_r_estimates(scenarios):
     for scenario in scenarios:
         scenario.get_r_estimate()
+
+def get_r_all_estimates(scenarios):
```
Comment (microsoft-fevieira) [inline estimators/test/utils.py:51]: Should we call this overall or aggregated?

**Review by microsoft-fevieira — APPROVED** [review]

File: estimators/test/utils.py (hunk 1 above)
Comment (ataymano) [reply estimators/test/utils.py:51]: do you mean everywhere or just in this helper file?

Comment (ataymano) [reply estimators/test/utils.py:51]: We already have get_r and get_r_given_impression, so I feel keeping get_r_* pattern is good for consistency. How about get_r_overall?

Comment (microsoft-fevieira) [reply estimators/test/utils.py:51]: sounds good to me

**Pull request merged**